cves/CVE-2010-3718.yml

```diff
@@ -64,8 +72,8 @@ fixes:
    - commit:
```

Comment:
What did this commit do exactly that made it a fix?

Reply:
Although the 'note' field is supposed to be used to describe how the fix was discovered, I went ahead and added what the commit did instead. The field was already populated beforehand when the file was generated, so I didn't find the commit. I suppose describing the contents of the commit is the next best thing. Thanks.